web/docs/operators/subscribe.md

```diff
@@ -21,6 +21,12 @@ The `subscribe` operator subscribes to events from a channel with the specified
 topic. Multiple `subscribe` operators with the same topic receive the same
 events.
 
+Subscribers propagate back pressure to publishers. If a subscribing pipeline
+fails to keep up, all publishers will slow down as well to a matching speed to
+avoid data loss. This mechanism is disabled for pipelines that are not visible
+on the overview page on app.tenzir.com, which drop data rather than slow down
+their publishers.
```

Comment:
```suggestion
on the overview page on [app.tenzir.com](https://app.tenzir.com), which drop data
rather than slow down their publishers.
```

Otherwise it's not shown as a link (you could also change the text if you want to, not sure how consistent we are when referencing the app).